### PR TITLE
Environment setup doc

### PR DESCRIPTION
**Pull request by lucasasecas**
Was generated the documentation to setup the project in a local environment and run it

### CODE REVIEW

File: src/_tsd.d.ts~ [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/// <reference path="../typings/knockout/knockout.d.ts" />
```
Comment (andresmoschini) [inline src/_tsd.d.ts~]: This file should not be here right?

File: src/_tsd.d.ts [hunk 2]
```diff
@@ -1,3 +1,3 @@
 /// <reference path="../typings/knockout/knockout.d.ts" />
-/// <reference path="../node_modules/definition-header/dist/index.d.ts" />
+/// <reference path="../node_modules/tsd/node_modules/definition-header/dist/index.d.ts" />
```
Comment (andresmoschini) [inline src/_tsd.d.ts]: This change is part of #1 pull request, right?

Comment (lucasasecas) [inline src/_tsd.d.ts]: > This change is part of #1 pull request, right?

Yes, I made the mistake to push with an old  commit.
I have to undo this commit and re-push right?

Comment (andresmoschini) [inline src/_tsd.d.ts]: Yes, that would be nice.

**Comment by lucasasecas** [comment]
was reverted commit 0f30faa.

**Comment by andresmoschini** [comment]
I though that it was more difficult to you to install it,  for example does it work with any node version? 

Following these simple steps anybody could make it works? 

If yes, go ahead and merge it.

**Comment by lucasasecas** [comment]
Although several solutions were implemented, the root problem that cause all issues was the file _tds.d.ts and the impossibility to rewrite that caused by the used command **tsd install**.
Solved that problem it's not necessary install a different version of nodejs.
It was tested on three different PCs and no problems found .

However, if any user has a problem we can update the file.

**Comment by andresmoschini** [comment]
Great! :shipit:

**Pull request merged**